src/panels/config/application_credentials/dialog-add-application-credential.ts

```diff
@@ -107,19 +110,31 @@ export class DialogAddApplicationCredential extends LitElement {
                 ${this.hass.localize(
                   "ui.panel.config.application_credentials.editor.missing_credentials",
                   {
-                    integration: domainToName(
-                      this.hass.localize,
-                      this._domain!
-                    ),
+                    integration: selectedDomainName,
                   }
                 )}
+                <a
+                  href=${documentationUrl(
+                    this.hass,
+                    `/integrations/${this._domain}`
```

Comment:
Should we account for custom integrations here?

Reply:
Is there a way to grab the integration documentation url from the manifest?